src/styles/blocks/button-link.css

```diff
@@ -0,0 +1,19 @@
+.button-link {
+  color: #fff;
+  text-transform: uppercase;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background-color: var(--blue-accent);
+  border: 1px solid var(--blue-accent);
+  font-weight: 700;
+  font-size: 14px;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.button-link:hover {
+  background-color: #fff;
+  color: var(--blue-accent);
+  border: 1px solid var(--blue-accent);
+  transition: all 0.1s ease-in-out;
```

Comment:
don't use `all` - list properties you really change - it's better for performance
also, I'd recommend setting at least 0.3s. 0.1s is too fast